trunk/source/PEAtoBoogie/src/de/uni_freiburg/informatik/ultimate/pea2boogie/translator/ReqSymboltableBuilder.java

```diff
@@ -76,6 +76,7 @@
 public class ReqSymboltableBuilder {
 
 	private static final BoogieLocation DUMMY_LOC = new BoogieLocation("", -1, -1, -1, -1);
+	private static final Boolean SUPPORT_HISTORY_VARS = false;
```

Comment:
There should be only one place where we can set this flag. Move it, e.g., to preferences.